gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestMapper.java

```diff
@@ -25,7 +25,7 @@
 import org.springframework.stereotype.Service;
 
 import lombok.AllArgsConstructor;
-
+]
```

Comment:
Martin, I think this causes a failure and needs to be removed.

Reply:
Yes, that has been removed there, that was left over in error when clearing unused exports / testing scoping.